src/modules/InviteUser/components/invitationActions.vue

```diff
@@ -47,7 +66,35 @@ export default {
               //error message in a dialog box
             }
         },
-    },
+        async deleteMember(){
+          try{
+            // const response = await deleteUser(this.id);
+            // if (response.data.status === true){
+              const response = await axios.delete(
+          `https://6319958e8e51a64d2be7568b.mockapi.io/invitedUsers/${this.id}`   // used only for testing
+        );
+              this.$emit('deleteInvite')
+            // }
+            //show success message
+          }
+          catch(error){
+            //error message
```

Comment:
Show an error message to the user, at least with "alert()", do not leave it empty.

Reply:
Toast message has been implemented in a seperate card